pybamm/experiment/experiment.py

```diff
@@ -39,7 +39,7 @@ class Experiment:
 
     def __init__(
         self,
-        operating_conditions: list[str],
+        operating_conditions: list[str | tuple[str]],
```

Comment:
should this be an `collections.Iterable` type?

Reply:
I don't think so, 
```
for cycle in operating_conditions:
            if not isinstance(cycle, tuple):
                cycle = (cycle,)
```
suggests that it's looking specifically for a tuple, not just any Sequence?